src/ResourceManager/Network/Commands.Network/RouteTable/NewAzureRMRouteTableCommand.cs

```diff
@@ -84,6 +89,7 @@ public override void Execute()
             {
                 Location = this.Location,
                 Routes = this.Route,
+                DisableBgpRoutePropagation = this.DisableBgpRoutePropagation
```

Comment:
@neaggarwMS for consistency, use `this.DisableBgpRoutePropagation.IsPresent` to determine whether or not the switch parameter was set